build/common.legacy.props

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <IsNetCoreProject>false</IsNetCoreProject>
   </PropertyGroup>
+  <ItemGroup Condition="'$(Shipping)' == 'true' AND '$(IsXPlat)' != 'true'">
```

Comment:
This would tag all projects though, woudln't it?

Reply:
Wouldn't this get added through legacy.props -> common.project.props -> common.props

Reply:
no..common.props is only imported for sdk projects... common.project.props is imported everywhere

common.legacy.props is imported for legacy projects

common.legacy.props and common.props import common.project.props , not the other way around.

Reply:
Actually this doesn't work. 
Totally slipped my mind. 

These conditions are not respected and all projects will be tagged with pdbgit. 

Don't know if that's a problem. 
Let's discuss offline.